### DISCUSSION TITLE
how to uninstall tpot?

### DISCUSSION THREAD

**Discussion by techstartupexplorer** [telekom-security/tpotce#1155] [Q&A]
hello,
how to uninstall tpot? installed it in Debian 11 thru digitalocean 2x and it doesn't work.. i can access the main page but when accessing kibana link it doesn't load up and checking the status of the honeypot its always DOWN. maybe the resource of the server is not enough to handle tpot.

NAME                  STATUS                       PORTS
fatt       Up 6 minutes   
nginx      Up 6 minutes   
p0f        Up 6 minutes   
suricata   Up 6 minutes   
adbhoney              DOWN                  
ciscoasa              DOWN                  
citrixhoneypot        DOWN

**Comment by t3chn0m4g3** [comment]
There is no uninstaller.

For troubleshooting check this thread #1082.

**Reply by techstartupexplorer** [reply]
docker kill $(docker ps -q)
docker rm $(docker ps -a -q)
docker rmi $(docker images -q)

deleted all docker images instead. The objective of the project is good but deployment is not yet stable.

**Reply by t3chn0m4g3** [reply]
As mentioned in the documentation, it is expected and known that some cloud offerings are not supported and that some modifications are necessary on the user's end to solve this.

For Digitial Ocean this has been discussed numerous times without users providing enough information to identify or even solve the issues causing the problems. T-Pot is community based and can only improve as good or as much the community is willing to support.

The provided steps do not uninstall T-Pot as the installer makes numerous changes to the system.
